### DISCUSSION TITLE
How to disable parentheses being added when importing react functional component?

### DISCUSSION THREAD

**Discussion by alivault** [hrsh7th/nvim-cmp#1855] [Q&A] [answered]
For example, if I have the following React component:

```jsx
export function Example() {
  return <h1>Example</h1>;
}
```

When I type `<Example` and hit enter, I get `<Example()`. The auto-completion recognizes the imported value as a function, but adds parentheses after it. I think the default should be not to call the function when importing.

**Comment by valerius21** [comment]
+1

**Reply by alivault** [reply, marked as answer]
I found a solution by disabling `completeFunctionCalls` in the tsserver settings. Explained here: https://www.aliabbas.dev/articles/troubleshooting-neovim-fixing-unwanted-parentheses-in-react-imports
